templates/web/base/admin/open311-form-fields.html

```diff
@@ -65,6 +76,21 @@
         <label for="send_comments" class="inline">[% loc('Use Open311 update-sending extension') %]</label>
     </p>
 
+    <div class="admin-hint">
+      <p>
+        [% loc(
+            "Enable <strong>Open311 problem-fetching</strong> if you want to display reports created at
+          the endpoint to FixMyStreet. If you're not sure, it probably does not, so leave this unchecked.
```

Comment:
I'd change this text to "Enable Open311 problem fetching if you want to display reports created at the endpoint on FixMyStreet. If you're not sure, you probably do not, so leave this unchecked."